### ISSUE TITLE
Automatically Updating when First Downloaded (sometimes)

### ISSUE THREAD

**Issue by mpsparrow** [easy-updates-manager/easy-updates-manager#45]
There appears to be some sort of error that automatically updates everything on peoples websites when the plugin is first installed.

https://wordpress.org/support/topic/theme-updated-despite-being-disabled?replies=12
https://wordpress.org/support/topic/pretty-cool-but?replies=3#post-

@easy-updates-manager/team: Do you think this is an error with the plugin or with something on there install?

**Comment by ronalfy** [comment]
I'll try to test this tomorrow, but I'm moving into an apartment this
weekend, so it will be difficult.

**Comment by mpsparrow** [comment]
I will do some testing also.

**Comment by mpsparrow** [comment]
Another one here https://wordpress.org/support/topic/theme-updated-despite-auto-updates-being-disabled?replies=2

**Comment by ronalfy** [comment]
Okay, I did some testing on this and was not able to duplicate it.

The first thing I did was delete the plugin and delete the options it creates.

I installed the plugin fresh. When the plugin is first installed, it does NOT save any options. It leaves WordPress alone. 

Only when I went and saved options was the plugin brought to life. Then it allowed/disallowed automatic updates.

My conclusion is this: the plugin, freshly installed, does not save any options or alter WordPress' default behavior whatsoever.  Any auto-updates are caused by something else, not by this plugin.
